customWorldboss.lua

```diff
@@ -731,9 +732,97 @@ Config_bossYellPhase2[7] = "Sssssszzzzzzzzz!"
 -- yell for the boss when they cast on themself
 Config_bossSpellSelfYell[7] = "SSSSSSSUFfffrrrrr!"
 
+------------------------------------------
+-- Begin of encounter 8 config
+------------------------------------------
+
+-- Database NPC entries. Must match the associated .sql file
+Config_bossEntry[8] = 1112071           --db entry of the boss creature
+Config_npcEntry[8] = 1112072            --db entry of the NPC creature to summon the boss
+Config_addEntry[8] = 1112073            --db entry of the add creature
+Config_npcText[8] = 91118                --gossip in npc_text to be told by the summoning NPC
+
+-- list of spells:
+Config_bossSpell1[8] = 35759            --directly applied to the tank-- instant fire damage + dot
+Config_bossSpell2[8] = nil              --on a random player within Config_bossSpell2MaxRange
+Config_bossSpell2MaxRange[8] = 50       --max range im m/y to check for targets for boss spell 2 (default 35)
+Config_bossSpell3[8] = 69528            --on the 2nd nearest player within 30m -- deals damage to target and damage to nearby additional target
+Config_bossSpell4[8] = nil              --on a random player within Config_bossSpell4MaxRange
+Config_bossSpell4Counter[8] = 10        --amount of casts to perform for spell 4. defaults to 1
+Config_bossSpell4MaxRange[8] = 30       --max range im m to check for targets for boss spell 4 (default 40)
+Config_bossSpell5[8] = nil              --directly applied to the tank with adds alive 
+Config_bossSpell6[8] = 70866            --directly applied to the tank when adds are dead --shadow blast (knockback + movement speed reduce)
+Config_bossSpell7[8] = 46469            --directly applied to the tank --melt armor 2000 for 30 secs
+Config_bossSpell8[8] = nil              --directly applied to the tank x seconds after spell 7 -
+Config_bossSpell8delay[8] = nil         --delay between spell 7 and 8. Must be smaller than timer7 / 2
+Config_bossSpellSelf[8] = 13022         --cast on boss while adds are still alive fire+arcane reflect
+Config_bossSpellEnrage[8] = 54356       --cast on boss once after Config_bossSpellEnrageTimer ms have passed-- Soft Enrage
+
+Config_bossSpellTimer1[8] = 10000       -- This timer applies to Config_bossSpell1
+Config_bossSpellTimer2[8] = 37000       -- This timer applies to Config_bossSpell2
+Config_bossSpellTimer3[8] = 12000       -- This timer applies to Config_bossSpellSelf in phase 1 and Config_bossSpell3+4 randomly later
+Config_bossSpellTimer5[8] = 20000       -- This timer applies to Config_bossSpell5+6
+Config_bossSpellTimer7[8] = 40000       -- This timer applies to Config_bossSpell7+8 (in ms)
+Config_bossSpellEnrageTimer[8] = 90000
+Config_minPhaseForTimer7[8] = 3         -- From this phase on the boss will use Timer 7 to cast Spell 7+8. Will ignore it before.
+
+Config_bossSpellModifier1bp0[8] = 5000     -- should be instant damage ??? @55Honey
```

Comment:
Yes, I think so, too. These values are before effects though. So they need to be tested, e.g. on the PTR.

Reply:
yeah I'm down to do that if you're down to bring the npc back on the PTR

Reply:
The "Amber Haze" NPC near Gadgetzan can only spawn 5man encounters, hence a single add.
But I'll happily spawn a Chromie so everyone can try it.

Reply:
yeah that would be nice :)